docs/user-guide/release-notes/2.3.x.rst

```diff
@@ -0,0 +1,12 @@
+2.3 Release Notes
+=================
+
+2.3.0
+-----
+
+See the :fixedbugs:`2.3.0`.
+
+A new command, docker repo tag, can point tags to manifests. It accepts
+`--repo-id`, `--tag-name`, and `--manifest-digest`. If a tag with the name
+does not exist, it will be created. The tag is then updated and pointed to the
```

Comment:
Good eye Ina.

s/The tag is then updated/If the tag exists, then it is updated/